### ISSUE TITLE
Onboarding: WC > Settings > Payments

### ISSUE THREAD

**Issue by v18** [Automattic/woocommerce-services#1482]
In 4.0, there's a big list of payment methods added, we should add better description for each of these payment methods.

cc @bmccotter - can you add your screenshot of that page here?

**Comment by bmccotter** [comment]
Absolutely! See here, specifically for the various methods Stripe supports

<img width="1110" alt="screen shot 2018-08-09 at 4 27 01 pm" src="https://user-images.githubusercontent.com/41127493/43908913-18d0befe-9bf1-11e8-8bbb-9876a24e418a.png">
